### DISCUSSION TITLE
0.3.0

### DISCUSSION THREAD

**Discussion by antoniospneto** [Engenere/BrazilFiscalReport#16] [Announcements]
## What's Changed
* Refatoração e Melhoria do Gerador da DANFE by @antoniospneto in https://github.com/Engenere/BrazilFiscalReport/pull/13
* Update readme by @antoniospneto in https://github.com/Engenere/BrazilFiscalReport/pull/14
* DANFE: Melhoria para o Simples Nacional by @antoniospneto in https://github.com/Engenere/BrazilFiscalReport/pull/15

**Full Changelog**: https://github.com/Engenere/BrazilFiscalReport/compare/0.2.1...0.3.0

<hr /><em>This discussion was created from the release <a href='https://github.com/Engenere/BrazilFiscalReport/releases/tag/0.3.0'>0.3.0</a>.</em>

**Comment by davisfjob** [comment]
Boa noite, tudo bem?

Gostaria de tirar uma dúvida sobre o uso da APi, a minha necessidade é somente fazer a leitura do código de barras (chave de acesso) da DANFE, e com isso baixar o XML, após tratar o XML o sistema faria a geração do PDF para guardar no banco de dados, é possível?

Muito obrigado.

**Comment by antoniospneto** [comment]
Oá @davisfjob,

A parte de gerar a DANFE a partir do arquivo XML, sim, você consegue fazer utilizando esta biblioteca, mas somente essa parte.
Quanto a parte de buscar o XML da nota utilizando os webservice do governo, não, essa lib não faz, isso você pode ver em outras bibliotecas, de uma olhada nessas: https://github.com/akretion/nfelib https://github.com/erpbrasil/erpbrasil.edoc

**Reply by davisfjob** [reply]
Boa Noite,  muito obrigado
